packages/spacecat-shared-data-access/src/models/importer/import-job.js

```diff
@@ -188,5 +189,9 @@ export const createImportJob = (data) => {
     throw new Error(`Invalid options: ${newState.options}`);
   }
 
+  if (!isObject(newState.initiatedBy)) {
```

Comment:
Does this mean we will need to remove all import jobs which do not have this property before merging this change? Should `initiatedBy` be optional?